external/anomaly/constraints.txt

```diff
@@ -17,4 +17,4 @@ requests==2.26.0
 scikit-image==0.17.2
 scikit-learn==0.24.2
 torchmetrics==0.6.0
-protobuf==3.20.0
+protobuf<3.21.0
```

Comment:
We are introducing conflicting changes in our PR. https://github.com/openvinotoolkit/training_extensions/pull/1124/files I'll update my PR to reflect your changes.

Reply:
I see. I'll put it back as previous.
For second comment, as far as I know, constraints file literally just constrains package version during install.
So it doesn't make some package installed like requirements.txt. Please refer this [docs](https://pip.pypa.io/en/stable/user_guide/#constraints-files).